spec/controllers/ansible_playbook_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe AnsiblePlaybookController do
+
+  include CompressedIds
+
+  before do
+    @playbook = FactoryGirl.create(:embedded_playbook)
+    EvmSpecHelper.create_guid_miq_server_zone
+    login_as FactoryGirl.create(:user_admin)
+  end
+
+  context "#show" do
+    #let(:playbook) { FactoryGirl.create(:embedded_playbook) }
+    subject do
+      get :show, :params => {:id => @playbook.id}
+    end
```

Comment:
Can be replaced with `subject { get :show, :params => {:id => playbook.id} }` as it short line. Meaning is same but code is shorter :)